client/js/main.js

```diff
@@ -0,0 +1,120 @@
+document.getElementById("velocity-time").width = document.getElementById("chart1").offsetWidth;
```

Comment:
Instead of writing `document.getElementById("velocity-time")` everywhere, you can set it to a variable instead!

```
let velocityTimeChart = document.getElementById("velocity-time");
let velocityChartDiv= document.getElementById("chart1");

velocityTimeChart.width = velocityChartDiv.offsetWidth;
```